package.json

```diff
@@ -3,14 +3,14 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "clean": "lerna clean",
-    "updated": "lerna updated",
     "diff": "lerna diff",
-    "publish": "lerna publish",
-    "dev": "lerna publish --npm-tag=dev --skip-git",
-    "rc": "lerna publish --npm-tag=rc --skip-git",
-    "force": "lerna publish --force-publish=*"
+    "updated": "lerna updated",
+    "release": "lerna publish",
+    "release-candidate": "lerna publish --npm-tag=rc",
```

Comment:
FYI keeping the `--skip-git` on `release-candidate` keeps the version changes locally but doesn't publish them, so when you go from release candidate to a release you don't have to force publish. If we don't do this then we'll need to add a force publish option to the scripts which I'm also not sure why you've removed in this pr. It also means we have more force publish options to consider as right now the forma script publishes every single package which we might not want to do.